src/sage/calculus/calculus.py

```diff
@@ -1189,15 +1206,80 @@ def limit(ex, dir=None, taylor=False, algorithm='maxima', **argv):
 
         sage: x = var('x')
         sage: f = (1 + 1/x)^x
-        sage: f.limit(x=oo)
+        sage: limit(f, x=oo)
+        e
+        sage: limit(f, x, oo) 
         e
         sage: f.limit(x=5)
         7776/3125
+        sage: f.limit(x, 5)
+        7776/3125
+
+    The positional `limit(expr, v, a)` syntax is particularly useful when
+    the limit variable `v` is an indexed variable or another expression
+    that cannot be used as a keyword argument (fixes :issue:`38761`)::
```

Comment:
```suggestion
    The positional ``limit(expr, v, a)`` syntax is particularly useful
    when the limit variable ``v`` is an indexed variable or another
    expression that cannot be used as a keyword argument
    (fixes :issue:`38761`)::
```

When referring to code, typeset it with two ticks (``code``) rather than one tick (`LaTeX`).

Reply:
You can see here how the formatting will look in the documentation: https://doc-pr-39812--sagemath.netlify.app/html/en/reference/calculus/sage/calculus/calculus#sage.calculus.calculus.limit